src/D2L.Bmx/Program.cs

```diff
@@ -199,7 +199,7 @@
 
 var updateCommand = new Command( "update", "Updates BMX to the latest version" );
 updateCommand.SetHandler( ( InvocationContext context ) => {
-	var handler = new UpdateHandler( new GitHubClient() );
+	var handler = new UpdateHandler( new GitHubClient(), new VersionProvider() );
```

Comment:
use DI as this provides "ambient" data that typically would be mocked in unit tests (if we ever add them)